.github/workflows/upnext.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  issues:
+    types: [labeled]
+jobs:
+  Move_Labeled_Issue_On_Project_Board:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: konradpabjan/move-labeled-or-milestoned-issue@v2.0
+      with:
+        action-token: "${{ secrets.GHA_UPNEXT_TOKEN }}"
```

Comment:
Is the token name quite specific and maybe we need something more generic for other actions?

Reply:
The token is deliberately specific as it has fine grained read only permissions just for this task. Other actions will use a different token according to the type of action and required privileges.